app/models/solr_document.rb

```diff
@@ -31,6 +31,7 @@ class SolrDocument
   attribute :num_collections, Solr::String, solr_name('num_collections', :stored_sortable, type: :integer)
   attribute :num_works, Solr::String, solr_name('num_works', :stored_sortable, type: :integer)
   attribute :holding_location, Solr::String, solr_name('holding_location')
+  attribute :rights_note, Solr::String, solr_name('rights_note')
```

Comment:
@andjsmit if single-valued,this needs a 2nd argument, such as 
Solrizer::Descriptor.new(:text, :stored, :indexed)
or
Solrizer::Descriptor.new(:text_en, :stored, :indexed)

to produce a suffix of _tsi or _tesi instead of _tesim

Reply:
Ok - Leaving as is.